accel/tcg/cputlb.c

```diff
@@ -1548,8 +1575,9 @@ static int probe_access_internal(CPUArchState *env, target_ulong addr,
         elt_ofs = offsetof(CPUTLBEntry, addr_read);
         break;
     case MMU_DATA_STORE:
-        elt_ofs = offsetof(CPUTLBEntry, addr_write);
-        break;
+        // CAP_STORE still returns the write offset so the data store can be
```

Comment:
Do we need to handle `MMU_DATA_CAP_LOAD` here?

Reply:
No, we do standard MMU_DATA_LOAD for cap loads, and leave it up to the caller to check PTE bits because traps are data dependant. Could also add a case for MMU_DATA_CAP_LOAD and assert not reached.